frontend/src/domain/entities/regulatory.js

```diff
@@ -31,6 +32,52 @@ export const mapToRegulatoryZone = properties => {
   }
 }
 
+const mapToFishingPeriodObject = fishingPeriod => {
+  if (fishingPeriod) {
+    const {
+      authorized,
+      annualRecurrence,
+      dateRanges,
+      dates,
+      weekdays,
+      holidays,
+      daytime,
+      timeIntervals,
```

Comment:
Pareil (`timeInterval`) - je chipote un peu :)

Reply:
Il y a plusieurs intervalles d'heures aussi.
Est-ce qu'on préfère timeIntervalList ?

Reply:
Ah ok, perso `timeIntervals` me convient dans ce cas, as you want!